ui/src/app/store/machine/types.ts

```diff
@@ -40,6 +40,31 @@ export type Machine = BaseNode & {
   zone: ModelRef;
 };
 
+export type MachineStatuses = {
+  [x: number]: {
```

Comment:
The machine system_id is a string isn't it? I wonder if in this case `[system_id: string]` might be more explanatory

Reply:
Ah yes, you're right. I wanted to do `Machine["system_id"]` there, but it wouldn't allow it.